gen.go

```diff
@@ -471,9 +478,9 @@ func NewObjectPropertyMappingValue(base string, named *types.Named, name string,
 				exportedFieldType: f.Type(),
 			}
 			// proto generated Getter
-			getter, ok := GetMethod(named, "Get"+name, ignoreCase)
-			if ok && getter.Exported() {
-				ret.getter = getter
```

Comment:
This block here is what's causing the bug. Adding a getter here indirectly changes the return value on type checks later on from [this](https://github.com/RoryQ/sesame/blob/development/gen.go#L542) method. Because of this, if the destination was a pointer, the `Type()` call would return the underlying type and not a pointer type which would cause sesame to handle it differently and generate code that didn't properly handle the pointers.